docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+#MySQL container 
+
+dbsql:
+    restart: always
+    image: mysql:5.6
+    ports:
+        - "3306:3306"
+    environment:
+        MYSQL_ROOT_PASSWORD: changeme
+        MYSQL_USER: flis
+        MYSQL_PASSWORD: flis
+        MYSQL_DATABASE: flis
+
```

Comment:
As there will be three databases (hstool, livecat and metadata), this flis database won't be used at all.

Is this a limitation of the mysql image? Can we simply get rid of them? Maybe provide instructions in the README about how to create the three databases?